bin/ghe-restore

```diff
@@ -178,6 +178,9 @@ if [ "$GHE_VERSION_MAJOR" -ge 2 ]; then
 fi
 
 echo "Restoring Git repositories ..."
+# Remove temporary 2.2 storage migration directory if it exists
+echo "if [ -d /data/user/repositories-nw-backup ]; then sudo rm -rf /data/user/repositories-nw-backup; fi" |
```

Comment:
We can axe the `if` here too. `rm -rf` will exit non-zero when the directory doesn't exist.